pkg/config/polling_manager_test.go

```diff
@@ -146,6 +150,129 @@ func TestNewAsyncPollingProjectConfigManagerWithNullDatafile(t *testing.T) {
 	mockRequester.AssertExpectations(t)
 }
 
+func TestGetConfigForPollingConfigManagerWaitsForTimeoutForInvalidDatafile(t *testing.T) {
+	blockingTimeout := 50 * time.Millisecond
```

Comment:
I suggest reducing this to 1 millisecond, combined with my other suggestion, there is no need for this to be so high.